kubetest2-gke/deployer/deployer.go

```diff
@@ -354,11 +473,7 @@ func (d *deployer) TestSetup() error {
 // Kubeconfig returns a path to a kubeconfig file for the cluster in
 // a temp directory, creating one if one does not exist.
 // It also sets the KUBECONFIG environment variable appropriately.
-func (d *deployer) Kubeconfig() (string, error) {
-	if err := d.prepareGcpIfNeeded(); err != nil {
-		return "", err
-	}
-
+func (d *deployer) Kubeconfig(project, cluster string) (string, error) {
```

Comment:
This is breaking the deployerwithkubeconfig interface.
The kubeconfig returned here should be a comma separated list of all the clusters created.

Reply:
It looks `kubetest2-gke` does not implement the `DeployerWithKubeconfig` interface, which defines `Kubeconfig() (string, error)` instead of `Kubeconfig(project, cluster string) (string, error)`?

Ref: https://github.com/kubernetes-sigs/kubetest2/blob/master/pkg/types/types.go#L86

Reply:
yeah it was previously, the current PR changes that behavior.
Also correction they should be `:` separated not comma separated
https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#linux

Reply:
Fixed the function signature, and used the correct separator for multi kubeconfig files - `:` for linux and MAC, `;` for windows.